_includes/v20.1/app/spring-data-jdbc/PagedAccountRepositoryImpl.java

```diff
@@ -0,0 +1,25 @@
+package io.roach.data.jdbc;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.springframework.transaction.annotation.Propagation.MANDATORY;
+
+@Repository
+// @Transactional is not needed but here for clarity since we want repos to always be called from a tx context
```

Comment:
// @Transactional annotation here to emphasise that repositories should always be called within an existing transaction context

Reply:
Updated.